scripts/vim-patch.sh

```diff
@@ -175,6 +175,7 @@ assign_commit_details() {
   vim_commit_url="https://github.com/vim/vim/commit/${vim_commit}"
   vim_message="$(git -C "${VIM_SOURCE_DIR}" log -1 --pretty='format:%B' "${vim_commit}" \
       | sed -e 's/\(#[0-9]\{1,\}\)/vim\/vim\1/g')"
+  vim_author="$(git -C "${VIM_SOURCE_DIR}" log -1 --pretty='format:%an <%ae>' "${vim_commit}")"
```

Comment:
`Co-authored-by: ` could be moved to the format instead of repeating it above.
```suggestion
  vim_author="$(git -C "${VIM_SOURCE_DIR}" log -1 --pretty='format:Co-authored-by: %an <%ae>' "${vim_commit}")"
```

Reply:
I was not sure how much to put in this format argument, I actually tried to build the entire message with it, and it wasn't really readable, but that should be fine